conn.go

```diff
@@ -88,6 +89,8 @@ func getSessionBusAddress(autolaunch bool) (string, error) {
 }
 
 // SessionBusPrivate returns a new private connection to the session bus.
+//
+// Deprecated: use ConnectSessionBus instead.
```

Comment:
You can now refer to other functions so that they are hyperlinked in docs on sites like pkg.go.dev.

```suggestion
// Deprecated: use [ConnectSessionBus] instead.
```